database/controllers.js

```diff
@@ -0,0 +1,25 @@
+const connection = require('./index.js');
+
+module.exports.getSong = (songId) => {
+  const select = `songs.album,
+    songs.album_imageUrl AS albumImageUrl,
+    songs.background_color AS backgroundColor,
+    songs.duration,
+    artists.name AS artistName,
+    songs.song_added AS songAdded,
+    songs.tag,
+    songs.title,
+    sound_profiles.height,
+    sound_profiles.profile
+  `;
+  const queryString = `SELECT ${select} FROM songs INNER JOIN artists ON songs.id = ? `
+    + ' AND songs.artist_id = artists.id'
+    + ' INNER JOIN sound_profiles ON sound_profiles.song_id = songs.id';
+  return connection.query(queryString, songId);
+};
```

Comment:
are you doing '_' for database stuff and '.' for the react items? there seems to be slight inconsistency